ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js

```diff
@@ -1,73 +1,139 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import { useSelector } from 'react-redux';
 
 import { getMaximumGasTotalInHexWei } from '../../../../../shared/modules/gas.utils';
+import { PRIORITY_LEVELS } from '../../../../../shared/constants/gas';
 import { GasLevelIconMap } from '../../../../helpers/constants/gas';
 import { PRIMARY } from '../../../../helpers/constants/common';
 import {
   decGWEIToHexWEI,
   decimalToHex,
+  hexWEIToDecGWEI,
 } from '../../../../helpers/utils/conversions.util';
+import { getAdvancedGasFeeValues } from '../../../../selectors';
 import { toHumanReadableTime } from '../../../../helpers/utils/util';
 import { useGasFeeContext } from '../../../../contexts/gasFee';
 import { useI18nContext } from '../../../../hooks/useI18nContext';
 import I18nValue from '../../../ui/i18n-value';
 import InfoTooltip from '../../../ui/info-tooltip';
 import UserPreferencedCurrencyDisplay from '../../user-preferenced-currency-display';
 
-const EditGasItem = ({ estimateType, onClose }) => {
+import { useCustomTimeEstimate } from './useCustomTimeEstimate';
+
+const EditGasItem = ({ priorityLevel, onClose }) => {
   const {
     estimateUsed,
     gasFeeEstimates,
     gasLimit,
-    setEstimateToUse,
-    updateTransaction,
+    maxFeePerGas: maxFeePerGasValue,
+    maxPriorityFeePerGas: maxPriorityFeePerGasValue,
+    updateTransactionUsingGasFeeEstimates,
+    transaction: { dappSuggestedGasFees },
   } = useGasFeeContext();
   const t = useI18nContext();
+  const advanecGasFeeDefault = useSelector(getAdvancedGasFeeValues);
```

Comment:
Since the function is called `getAdvancedGasFeeValues` what do you think about naming this `advancedGasFeeValues`, so it's easier to track where it comes from when scanning the code? Or minimally, `advanec` should be `advanced`, I'd imagine.

Reply:
done